src/_components/NavBarComponent.jsx

```diff
@@ -1,35 +1,43 @@
-import React, { Component } from 'react';
-import jwt from 'jsonwebtoken';
-import { Dropdown, Menu, Icon } from 'semantic-ui-react';
-
-import '../_css/NavBarComponent.css';
-
-const NavBarComponent = (props) => {
-  const { title } = props;
-  const token = jwt.decode(localStorage.getItem('token'));
-  let username = '';
-
-  if (token != null) {
-    username = token.username;
-  }
-
-  return (
-    <div className='navbar'>
-      <Menu secondary stackable>
-        <Menu.Item name='menu' onClick={props.toggleVisibility}>
-          <Icon name='list layout'/>
-        </Menu.Item>
-        <Menu.Item name={title}/>
-        <Menu.Menu position='right'>
-          <Dropdown item text={username} icon='user' simple>
-            <Dropdown.Menu>
-              <Dropdown.Item>Logout</Dropdown.Item>
-            </Dropdown.Menu>
-          </Dropdown>
-        </Menu.Menu>
-      </Menu>
-    </div>
-  );
-};
-
-export default NavBarComponent;
+import React from 'react';
+import jwt from 'jsonwebtoken';
+import { Dropdown, Menu, Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/NavBarComponent.css';
+
+const NavBarComponent = (props) => {
+  const { title, toggleVisibility } = props;
+  const token = jwt.decode(localStorage.getItem('token'));
+
+  const username = token !== null ? token.username : '';
+
+  return (
+    <div className="navbar">
+      <Menu secondary stackable>
+        <Menu.Item name="menu" onClick={toggleVisibility}>
+          <Icon name="list layout" />
+        </Menu.Item>
+        <Menu.Item name={title} />
+        <Menu.Menu position="right">
+          <Dropdown item text={username} icon="user" simple>
+            <Dropdown.Menu>
+              <Dropdown.Item>Logout</Dropdown.Item>
+            </Dropdown.Menu>
+          </Dropdown>
+        </Menu.Menu>
+      </Menu>
+    </div>
+  );
+};
+
+NavBarComponent.propTypes = {
+  toggleVisibility: PropTypes.func,
+  title: PropTypes.string
+};
+
+NavBarComponent.defaultProps = {
+  toggleVisibility: true,

```

Comment:
toggleVisibility is a function so I don't think a boolean would be an appropriate default value. Maybe you can make toggleVisibility a required prop so you don't have to provide a default value